app/scripts/worker.js

```diff
@@ -71,7 +71,7 @@ function diff_prettyHtml(diffs, templateMatch, selection) {
       text = data.replace(pattern_amp, '&amp;')
                  .replace(pattern_lt, '&lt;')
                  .replace(pattern_gt, '&gt;')
-                 .replace(pattern_para, '&para;<br>');
+                 .replace(pattern_para, '<br>');
```

Comment:
I realize this is existing code, but there's a DRY problem here since you had to update it in like 4 locations. Let's refactor it with a helper function so you can do it just once.